.github/workflows/dummy-agent-test.yml

```diff
@@ -14,20 +14,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Free Disk Space (Ubuntu)
```

Comment:
Just curious - is this free disk space required, bc we are installing additional buildx?

Reply:
No, those are separate issues. But at the time I developed this, workflows were failing or getting stuck for over an hour and some digging showed a out of disc space error. It is worth the extra 2 minutes rather than bug hunting for hours in the long run.